### ISSUE TITLE
when next/previous ghost text suggestion is shown, we don't alert screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#186675]
1. trigger an inline ghost text suggestion
2. use `Alt+[ or ]` to get new suggestions
3. 🐛 not announced to screen reader users

cc @jooyoungseo
